src/pybind/mgr/status/module.py

```diff
@@ -336,8 +336,8 @@ def handle_osd_status(self, bucket: Optional[str] = None, format: str = 'plain')
                 stats = osd_stats[osd_id]
                 assert metadata
                 hostname = metadata['hostname']
-                kb_used = stats['kb_used'] * 1024
-                kb_avail = stats['kb_avail'] * 1024
+                kb_used = stats['kb_used']
```

Comment:
this change fixes the output of `ceph osd status --format json`, but it breaks the output in the plain text format, which format this field with `mgr_util.format_bytes()`. and this very function expects the parameter in bytes.